sql-plugin/src/main/scala/org/apache/spark/sql/rapids/RapidsShuffleInternalManagerBase.scala

```diff
@@ -805,7 +794,9 @@ abstract class RapidsShuffleThreadedReaderBase[K, C](
               }
             }
             // keep track of the overall metric which includes blocked time
-            fetchTime = System.nanoTime() - fetchTimeStart
+            val fetchTime = System.nanoTime() - fetchTimeStart
+            deserializationTimeNs.foreach(_ += (fetchTime - readBlockedTime))
```

Comment:
I'd remove `readBlockedTime` from here, since it's just 0.

Reply:
readBlockedTime can be incremented in the `while` code block above.

Reply:
ah never mind

Reply:
readBlockedTime is incremented in above loop, so it should still be required in this line